src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpExecutionServiceDAO.scala

```diff
@@ -1,23 +1,25 @@
 package org.broadinstitute.dsde.rawls.dataaccess
 
 import akka.actor.ActorSystem
+import akka.util.Timeout
 import org.broadinstitute.dsde.rawls.model._
 import org.broadinstitute.dsde.rawls.model.ExecutionJsonSupport._
 import org.broadinstitute.dsde.rawls.util.FutureSupport
-import scala.concurrent.{Future, Await}
-import scala.concurrent.duration.Duration
+import scala.concurrent.Future
+import scala.concurrent.duration.FiniteDuration
 import spray.client.pipelining._
 import spray.http.FormData
 import spray.httpx.SprayJsonSupport._
 
-import scala.util.{Failure, Success, Try}
+import scala.util.Try
 
 /**
  * @author tsharpe
  */
-class HttpExecutionServiceDAO( executionServiceURL: String )( implicit val system: ActorSystem ) extends ExecutionServiceDAO with DsdeHttpDAO with Retry with FutureSupport {
+class HttpExecutionServiceDAO( executionServiceURL: String, workflowSubmissionTimeout: FiniteDuration )( implicit val system: ActorSystem ) extends ExecutionServiceDAO with DsdeHttpDAO with Retry with FutureSupport {
```

Comment:
Uses a FiniteDuration instead of a Timeout because:
# getDuration returns a FiniteDuration
# I'm wary about having a Timeout in scope in case it gets applied as the implicit timeout for the whole DAO.  We only want it for submitWorkflow.